docs/core/tools/dotnet-nuget-sign.md

```diff
@@ -71,7 +71,12 @@ The `dotnet nuget sign` command signs all the packages matching the first argume
 
 - **`--certificate-fingerprint <FINGERPRINT>`**
 
-   SHA-1 fingerprint of the certificate used to search a local certificate store for the certificate.
+   Specifies the fingerprint of the certificate used to search a local certificate store for the certificate.
+
+  > [!NOTE]
+  > Starting with `.NET 9 Preview 7`, a `NU3043` warning is raised when a SHA-1 certificate fingerprint is passed.
```

Comment:
We're going to be out of preview here soon. I would suggest we just say "starting with .NET 9"

```suggestion
  > Starting with `.NET 9`, a `NU3043` warning is raised when a SHA-1 certificate fingerprint is passed.
```

Reply:
Thanks for the feedback. Updated the doc.